packages/rich-text/README.md

```diff
@@ -273,6 +273,10 @@ _Returns_
 
 -   `Object`: A new value with replacements applied.
 
+<a name="RichText" href="#RichText">#</a> **RichText**
+
+Undocumented declaration.
```

Comment:
@nosolosw Any idea how we can add the doc block from `RichText` here? I guess it's not caught because it goes through `compose`?

Reply:
Yeah, exactly. In this case, there are a couple of points where we may introduce the docblock:

- in the [export statement](https://github.com/WordPress/gutenberg/pull/15212/files#diff-e063d548f7e2e25c9d72ee1a9fdc1129R37) at the `src/index.js`
- in the [default export](https://github.com/WordPress/gutenberg/pull/15212/files#diff-b7dace988233716c1772730cd1700292R1108) at `src/component/index.js`.

Any of those will do. Those aren't ideal, but we don't have a good solution yet for documenting components that are wrapped with HOCs.

Reply:
Thank you. :)